pkg/upload/dsym.go

```diff
@@ -187,15 +188,17 @@ func ProcessDsym(
 			err = server.ProcessFileRequest(endpoint+"/dsym", uploadOptions, fileFieldData, timeout, retries, dsym.UUID, dryRun)
 
 			if err != nil {
-				if failOnUpload {
```

Comment:
We can't downgrade to WARN without this check, so the integration test is failing because of this